cmd/svcat/output/binding.go

```diff
@@ -123,6 +119,28 @@ func WriteAssociatedBindings(w io.Writer, bindings []v1beta1.ServiceBinding) {
 	t.Render()
 }
 
+// WriteAssociatedSecret prints the secret data associated with a binding.
+func WriteAssociatedSecret(w io.Writer, secret *v1.Secret, err error) {
+	// Don't print anything the secret isn't ready yet
+	if secret == nil && err == nil {
+		return
+	}
+
+	fmt.Fprintln(w, "\nSecret Data:")
+	if err != nil {
+		// We should have been able to find a secret but couldn't for some reason,
+		// warn about it without blowing up the entire command.
+		fmt.Fprintf(w, "  %s", err.Error())
+		return
+	}
+
+	t := NewDetailsTable(w)
+	for key, value := range secret.Data {
+		t.Append([]string{key, fmt.Sprintf("%d bytes", len(value))})
```

Comment:
is it a bad idea to show the secret data? Or output the kubectl command to display the data if we are unwilling to?

Reply:
I was just blindly ripping off kubectl but it would be great to have it do the decoding for you. Mind if I add that as a follow-on PR for the new flag?

Reply:
Here's an issue to track adding that flag

https://github.com/kubernetes-incubator/service-catalog/issues/1970